docs/apache-airflow-providers-influxdb/connections/influxdb.rst

```diff
@@ -16,34 +16,21 @@
     specific language governing permissions and limitations
```

Comment:
This is wrong...
you are altering `influxdb.rst` with Github content.
the `influxdb.rst` should not be modified by this PR.
You should create a new file `github.rst` in the correct path

Reply:
thanks for pointing it out, It looks like there has been a wrong paste.